docs/devel/development.md

```diff
@@ -339,6 +354,8 @@ hack/update-generated-docs.sh
 ```
 
 
+
```

Comment:
Nit: These spaces appear unnecessary. 

But don't change it, or lgtm will be removed again. :-/